doc/tools/docgram/orderedGrammar

```diff
@@ -81,6 +81,7 @@ term_application: [
 
 arg: [
 | "(" ident ":=" term ")"
+| "(" natural ":=" term ")"
```

Comment:
While this change to `orderedGrammar` is appropriate, you should add a line `| test_lpat_nat_coloneq` in `common_edit.mlg` just after the line `| test_lpar_id_coloneq`.  Otherwise the next time someone runs `make doc_gram_rsts`, the `test_lpat_nat_coloneq` will be added in here automatically (and incorrectly).